### ISSUE TITLE
Auto-set team break eligibility per speaker eligibility

### ISSUE THREAD

**Issue by tienne-B** [TabbycatDebate/tabbycat-frontend#12]
Within the Eligibility page, we could merge the Speaker and Team category pages, by having the speakers be expanded under their team. Then, there can be arrays of checkboxes for both speaker and break categories.

In the backend, we could add a preference "minimum speakers to render team eligible" and add a field linking speaker categories to break categories. Then, if _n_ speakers of a team get marked, the team will too. The team should be able to be unmarked thereafter. (Un-)Marking a team should not affect the speakers however.

**Comment by valpang22** [comment]
Can also be a format specific thing, BP/CP would 2 speakers in category = eligible as a default.
Unsure about other formats, but the preference to change minimum speakers is going to be very helpful

**Comment by tienne-B** [comment]
Yup! The preference can then be added to the CP/BP presets. The default if no preset is selected should be disabled though.
